<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetBugTrackerTaskDetailsList</class>
 <widget class="QDialog" name="GetBugTrackerTaskDetailsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get Bug Tracker Task Details List</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>BugTracker Details</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Bug Task Title :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bug Status                 :</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>BugTaskDescription</string>
    </property>
    <widget class="QTextEdit" name="textEditBugTaskDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBoxBugTaskTitle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditBugStatus">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
